docs/usage/sat_bootprep.md

```diff
@@ -516,20 +516,20 @@ Only one layer is shown for brevity.
 configurations:
 - name: "{{default.note}}compute-{{recipe.version}}{{default.suffix}}"
   layers:
-  - name: cos-compute-{{cos.working_branch}}
-    playbook: cos-compute.yml
+  - name: cne-compute-{{cne.working_branch}}
+    playbook: cne-compute.yml
```

Comment:
We should double-check that the name of the playbook is changing from `cos-compute.yml` to `cne-compute.yml`.